subscription.go

```diff
@@ -103,6 +103,12 @@ func (r *marathonClient) registerSubscription() error {
 	case EventsTransportCallback:
 		return r.registerCallbackSubscription()
 	case EventsTransportSSE:
+		if timeout := r.client.config.HTTPClient.Timeout; timeout > 0 {
```

Comment:
We still provide support for Go 1.5. We have run into a few bugs already that are only fixed in 1.6, so we might drop support for that earliest version. I'd rather not drop support for 1.6, however.